R/Intervention_file_generator.R

```diff
@@ -0,0 +1,147 @@
+
+generate_intervention_file <- function() {

```

Comment:
This function is going to write out the file `intervention_coverage.txt` into the current working directory: I think it should at least get a `path` argument to indicate the directory into which the file will be written

Reply:
Actually, this does not appear to be (yet) used

Reply:
I've made a note of this [here](https://github.com/mrc-ide/vivax/issues/3)